README.md

```diff
@@ -523,10 +523,6 @@ License: [MIT](https://github.com/taylorhakes/promise-polyfill/blob/master/LICEN
 Copyright &copy; 2013-present, Facebook, Inc.
 License: [MIT](https://github.com/facebook/prop-types/blob/master/LICENSE)
 
-[**react-broadcast**](https://github.com/ReactTraining/react-broadcast)  

```

Comment:
This PR might also remove some other dependencies that were pulled in by react-broadcast (can run `npx license-checker --production --json | jq 'map_values({ licenses, publisher, repository }) | del(.[][] | nulls)'` to check).